rem/aufile/aufile.c

```diff
@@ -278,3 +278,43 @@ size_t aufile_get_length(struct aufile *af, struct aufile_prm *prm)
 
 	return 0;
 }
+
+/**
+ * Set initial playing position of a WAV file in ms
+ *
+ * @param af  Audio-file
+ * @param prm Audio file parameters from aufile_open
+ *
+ * @return 0 if success, otherwise errorcode
+ */
+int aufile_set_position(struct aufile *af, struct aufile_prm *prm,
+						   off_t pos_ms)
+{
+	struct wav_fmt fmt;
+	int err;
+	off_t pos;
+	size_t datasize;
+
+	if (!af || !prm) {
+		return EINVAL;
+	}
```

Comment:
please remove curly braces that is not needed here ... same for the rest of the function